server/plugins/irc-events/link.ts

```diff
@@ -167,7 +175,7 @@ function parseHtmlMedia($: any, preview, client: Client): Promise<FetchRequest>
 				return;
 			}
 
-			$(`meta[property="og:${type}:type"]`).each(function (this: cheerio.Element, i: number) {
+			$(`meta[property="og:${type}:type"]`).each(function (this: Element, i: number) {
```

Comment:
nack, stop lying to the compiler.

Reply:
> nack, stop lying to the compiler.

Well, I gotta confess: I don’t know that the actual type needs to be. The problem with `cheerio.Element` is that it relies on `@types/cheerio`, right? And if we have `@types/cheerio`, then we can’t use `cheerio.loadBuffer`?

Reply:
remove the types from the .each() function.
Type the $ argument passed into the current scope (the top level function) as to what it is.

Cheerio has built in types, it'll figure it out correctly